zerver/management/commands/makemessages.py

```diff
@@ -50,7 +50,7 @@
 strip_whitespace_left = re.compile(u"\\s+(%s-\\s*(endtrans|pluralize).*?-?%s)" % (
                                    BLOCK_TAG_START, BLOCK_TAG_END), re.U)
 
-regexes = ['{{#tr .*?}}(.*?){{/tr}}',
+regexes = ['{{#tr .*?}}([\s\S]*?){{/tr}}',
```

Comment:
So by default '.' character does not match newline character `\n`. I think you should add a comment here.